CHANGELOG.md

```diff
@@ -102,12 +102,14 @@ The table below shows which release corresponds to each branch, and what date th
 - [#2281][2281] FIX: Getting right amount of data for search fix
 - [#2287][2287] Fix `_countdown_handler` not invoking `timeout_change`
 - [#2294][2294] Fix atexit SEGV in aarch64 loader
+- [#2352][2352] add `RETURN_CONST` as an allowed `_const_code` in safeeval
```

Comment:
Please add this to the 4.11.2 changelog above instead

Reply:
Oh right sorry, fixed!